gTea/app/src/main/java/com/nechaieva/gtea/OrderProcessor.java

```diff
@@ -0,0 +1,64 @@
+package com.nechaieva.gtea;
+
+import java.lang.reflect.Array;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class OrderProcessor {
+
+    List<String> menu;
+    final int SENSITIVITY = 3;
+    public final String NO_CANDIDATE = "";
+
+    OrderProcessor(String[] menu) {
+        this.menu = Arrays.asList(menu);
+    }
+
+    OrderProcessor(ArrayList<String> menu) {
+        this.menu = menu;
+    }
+
+    public String findInMenu(String query) {
+        Match match = findClosest(query);
+        return match.getDistance() < SENSITIVITY ? match.getMatchString() : NO_CANDIDATE;
```

Comment:
What about return an optional instead? This would make the "no result" case more expressive...